nrm_django/grants/models.py

```diff
@@ -207,6 +278,9 @@ class Meta:
         db_table = "ii_ga_authorities"
         verbose_name_plural = "Grant authorities"
 
+    def __str__(self):
+        return "%s: %s" % (self.authority_cd, self.grant)
```

Comment:
Would this be easier if it included authority_desc too? Some of the codes are pretty terse.

Reply:
I *think* we're OK, but if users say otherwise it's certainly easy enough to change!